packages/react-server-cli/src/compileClient.js

```diff
@@ -135,10 +135,15 @@ const packageCodeForBrowser = (entrypoints, outputDir, outputUrl, hot, minify, l
 					exclude: /node_modules/,
 				},
 				{
-					test: /.css$/,
+					test: /\.css$/,
 					loader: extractTextLoader,
 					exclude: /node_modules/,
 				},
+				{
+					test: /\.(eot|woff|woff2|ttf|ttc|png|svg|jpg|jpeg|gif|cgm|tiff|webp|bmp|ico)$/,
```

Comment:
Wow, that's a lot of file types!

Should this be `/.../i` to pick up upper-case extensions?